resources/leaflet/jquery.leaflet.js

```diff
@@ -336,7 +336,11 @@
 			this.bindClickTarget();
 
 			var layers = {};
-			$.each(options.layers.reverse(), function(index, layerName) {
+			var optionLayers = options.layers;
+			if ( sm.isUserUsesDarkMode() ) {
```

Comment:
I'm unsure why you put this function in semanticMaps.js. Could you move it into this file?

Reply:
😁 sure, actually, it was here but the last moment I decided it would be a better fit for `semanticMaps.js` - my idea was like it's something global and maybe could be used by other map services scripts in future